src/charm.py

```diff
@@ -120,44 +105,24 @@ def _update_fiveg_n3_relation_data(self) -> None:
 
     def _network_attachment_definitions_from_config(self) -> list[NetworkAttachmentDefinition]:
         """Returns list of Multus NetworkAttachmentDefinitions to be created based on config."""
-        access_nad_config = {
-            "cniVersion": "0.3.1",
-            "ipam": {
-                "type": "static",
-                "routes": [
-                    {
-                        "dst": self._get_gnb_subnet_config(),
-                        "gw": self._get_access_network_gateway_ip_config(),
-                    },
-                ],
-                "addresses": [
-                    {
-                        "address": self._get_access_network_ip_config(),
-                    }
-                ],
-            },
-            "capabilities": {"mac": True},
-        }
+        access_nad_config = self._get_access_nad_config()
         if (access_interface := self._get_access_interface_config()) is not None:
             access_nad_config.update({"type": "macvlan", "master": access_interface})
         else:
             access_nad_config.update({"type": "bridge", "bridge": "access-br"})
-        core_nad_config = {
-            "cniVersion": "0.3.1",
-            "ipam": {
-                "type": "static",
-                "addresses": [
-                    {
-                        "address": self._get_core_network_ip_config(),
-                    }
-                ],
-            },
-            "capabilities": {"mac": True},
-        }
+
+        if (access_interface_mtu := self._get_access_interface_mtu_config()) is not None:
+            access_nad_config.update({"mtu": int(access_interface_mtu)})
```

Comment:
This function will be called by the constructor, so I think we should validate the MTU before updating the NAD config.

Reply:
Done